packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -74,13 +78,22 @@ export interface RepeaterFieldsetProps {
   hideIndex?: boolean;
   /** Disables deleting, moving and editing fields */
   disabled?: boolean;
+  /** Locks item position (removes append and decrement actions) */
+  numOfLockedItems?: number;
   children: React.ReactNode;
+  /** Callback function when this item's index is changed' */
+  onMove: (from: number, to: number) => void;
+  /** Callback when the item is removed from the array */
+  onRemove: (index: number) => void;
+  /** Callback function when edit button is clicked */
+  onSave?: () => void;
```

Comment:
I think it would be helpful to pass the index here as well. Since this (at least to start) will be opening a dialog with a form, it would be useful to get the appropriate info from the index.

Reply:
Fixed by [352db51](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/352db510147821f32c820480e8561299b3f24f1c)